src/ssz/type/byte_vector.zig

```diff
@@ -41,6 +43,10 @@ pub fn ByteVectorType(comptime _length: comptime_int) type {
             try merkleize(@ptrCast(&chunks), chunk_depth, out);
         }
 
+        pub fn clone(value: *const Type, out: *Type) !void {
+            out.* = value.*;
```

Comment:
what's the difference between doing this and doing `@memcpy`? More curious as thats what i was expecting